src/battle_ai_util.c

```diff
@@ -499,6 +499,7 @@ s32 AI_CalcDamage(u32 move, u32 battlerAtk, u32 battlerDef, u8 *typeEffectivenes
     gBattleStruct->swapDamageCategory = FALSE;
     gBattleStruct->zmove.active = FALSE;
     gBattleStruct->zmove.baseMoves[battlerAtk] = MOVE_NONE;
+
```

Comment:
What's the reason for the addition of the newline here, given that you've removed a newline before a final return statement in other functions?

Reply:
accidentally

Reply:
hm, this might have been not accidental. I didn't like how was there no space between the return statement and the BattleStruct resets. Regardless I can remove the blank line.

Reply:
It's just another matter of inconsistencies in the repo, haha. I'm not opposed to just leaving it like this and cleaning it up in a follow-up PR (same should probably happen to pret)